pkg/endpoint/endpoint_test.go

```diff
@@ -0,0 +1,26 @@
+package endpoint
+
+import (
+	"context"
+	"testing"
+)
+
+func TestListen(t *testing.T) {
+	var config Config
+	var ctx context.Context
+	config.Listener = ""
+	endpoints := [5]string{
```

Comment:
```suggestion
	endpoints := []string{
```
The 5 is not needed.  Putting a number there creates an actual array and not a slice.  Arrays are using in go really only in very performance critical situations.